tests/conftest.py

```diff
@@ -49,11 +55,12 @@
 IS_LINUX = sys.platform.startswith("linux")
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture(autouse=have_blockbuster)
 def blockbuster(request: pytest.FixtureRequest) -> Iterator[None]:
-    # Allow selectively disabling blockbuster for specific tests
-    # using the @pytest.mark.skip_blockbuster marker.
-    if "skip_blockbuster" in request.node.keywords:
+    # Disable blockbuster if it is not available, and allow selectively
+    # disabling blockbuster for specific tests using the
+    # @pytest.mark.skip_blockbuster marker.
+    if not have_blockbuster or "skip_blockbuster" in request.node.keywords:
```

Comment:
This check is redundant, right? We should either be doing this check with autouse=True, or not doing this check.